contracts/helpers/sol/lib/OrderLib.sol

```diff
@@ -284,4 +288,120 @@ library OrderLib {
         advancedOrder.signature = order.signature;
         advancedOrder.extraData = extraData;
     }
+
+    /**
+     * @dev Converts Orders to AdvancedOrders in bulk.
+     *
+     * @param orders the Orders to convert
+     * @param numerator the numerator to set for all
+     * @param denominator the denominator to set for all
+     * @param extraData the extra data to set for all
+     *
+     * @return advancedOrders the AdvancedOrders
+     */
+    function toAdvancedOrders(
+        Order[] memory orders,
+        uint120 numerator,
+        uint120 denominator,
+        bytes memory extraData
+    ) internal pure returns (AdvancedOrder[] memory advancedOrders) {
+        for (uint256 i = 0; i < orders.length; i++) {
+            advancedOrders[i] = toAdvancedOrder(
+                orders[i],
+                numerator,
+                denominator,
+                extraData
+            );
+        }
+    }
+
+    /**
+     * @dev Converts an Order to an AdvancedOrders array.
+     *
+     * @param order the Order to convert
+     * @param numerator the numerator to set for all
+     * @param denominator the denominator to set for all
+     * @param extraData the extra data to set for all
+     *
+     * @return advancedOrders the AdvancedOrders
+     */
+    function toAdvancedOrders(
```

Comment:
personally more in favor of `SeaportArrays.AdvancedOrders(order.toAdvancedOrder())` than a dedicated lib fn for this – I still want to write a script to generate these libs and it'll need special logic for each non-generated struct-specific helper fn

Reply:
roger, will update